CHANGES.rst

```diff
@@ -12,17 +12,47 @@ The current role maintainer_ is ypid_.
 
 
 debops.apache v0.1.0 - unreleased
-----------------------------------------
+---------------------------------
 
 Added
 ~~~~~
 
 - Initial coding and design. [ypid_]
 
 - Add/Set the default `Referrer Policy`_ to ``no-referrer`` and made it
-  configurable via :ref:`item.http_referrer_policy <apache__ref_vhosts_http_referrer_policy>`.
+  configurable via :ref:`item.http_referrer_policy <apache__ref_vhost_http_referrer_policy>`.
   [ypid_]
 
+- Add the :envvar:`apache__mpm_max_connections_per_child` variable to allow to
+  configure the number of requests a child process should handle before
+  terminating. [ypid_]
+
+- Add support to enable and configure `Apache mod_status`_. You can set
+  :envvar:`apache__status_enabled` to ``True`` to enable it and make the status
+  page accessible from localhost. [ypid_]
+
+- Add support for generic Apache template generation using the `Apache IfVersion directive`_.
+  This feature can be configured by :envvar:`apache__config_use_if_version` and
+  :envvar:`apache__config_min_version`. [ypid_]
+
+- Add Ansible tags for env roles. To only prepare the Apache role
+  environment, you can use the ``role::apache:env`` tag. [ypid_]
+
+Changed
+~~~~~~~
+
+- Change default virtual host server name from ``000-default`` to ``default.{{
```

Comment:
Before you delete any files, check if they are configuration files included in a package:
```
dpkg-query -s apache2
dpkg -S /etc/apache2/sites-available/000-default
```
If so, you shouldn't remove them. This might hinder future upgrades, automated or not. Instead, you should divert the affected files either in the same directory with a different suffix (usually `.dpkg-divert`) or to another directory altogether so that the package manager still knows that they exist and can perform the package upgrades correctly.

This is different with the configuration files generated on the package installation dynamically, or managed by the `ucf` system. In this case, you should check the postinstall script of a given package for some clues how to handle this.